checklist.md

```diff
@@ -148,20 +115,18 @@ BAD example:
 }
 ```
 
-11. [SASS] - use variables for the main values so that you'll be able to reuse
-them and give them descriptive names. But don't overuse them, don't create
+10. [SASS] - use variables for the main values so that you'll be able to reuse
+them and **give them descriptive names**. But don't overuse them, don't create
 variable for the value that's used just once.
 
-12. [CODE STYLE] - Remember about styles properties order - ([css order](https://codeguide.academy/html-css.html#css-order))
-
-13. [LAYOUT] - don't hardcode max-width, your catalog container should work fine for any number of cards
-
-GOOD example:
-
-![Screenshot 2021-09-16 at 16 01 30](https://user-images.githubusercontent.com/32794096/133616959-0016cb9f-7083-4f93-8de6-f6a5c702e2ed.png)
```

Comment:
Why did you delete this point? It's super helpful, but it should be rephrased

Reply:
Let's add suggestion here like: "Use sass functions for dynamic `max-width` calculation"